index.rst

```diff
@@ -5,77 +5,331 @@
 Abstract
 ========
 
-The First-Look Analysis and Feedback Functionality (FAFF) working group, identified the need for a service that is capable of executing user-defined operations/computation following user-defined rules.
-This service has been referred  to as Catcher.
-In essence, the Catcher is not that different from the Watcher or other stream processing off-the-shelf technologies like Kapacitor.
-This technote explores the idea of using InfluxDB OSS 2.x new task engine as an alternative implementation for the Catcher.
+This technote contains design considerations for the Catcher, an event-driven processing component for the Rubin Observatory Control System.
+We briefly review the existing functionality of the system and discuss the use case for the Catcher.
+The idea originated from the First-Look Analysis and Feedback Functionality (FAFF) working group, and a more in-depth analysis of the use cases can be found in their reports.
+
 
 Introduction
 ============
 
 One of the finding of the First-Look Analysis and Feedback Functionality (FAFF) working group :sitcomtn:`025` :cite:`SITCOMTN-025`, was the need for a service to ...
 
-   handle the low-level coordiation of identifying when a specific condition is met, then launching and monitoring an analysis process.
+   handle the low-level coordination of identifying when a specific condition is met, then launching and monitoring an analysis process.
```

Comment:
I think this should be all on one line? Or maybe the PR just looks weird.

Reply:
What do eyes mean? Also, it still looks messed up... even in the raw code.
![image](https://user-images.githubusercontent.com/14132079/200077108-a6d31af3-5818-4ac6-8ae2-9e8c1cca0684.png)